schema/show.js

```diff
@@ -34,9 +34,11 @@ import {
   GraphQLUnionType,
 } from 'graphql';
 
-const kind = ({ artists, fair, artists_without_artworks }) => {
+const kind = ({ artists, fair, artists_without_artworks, group }) => {
   if (isExisty(fair)) return 'fair';
-  if (artists.length > 1 || (artists_without_artworks && artists_without_artworks.length > 1)) {
+  if (group ||
+      artists.length > 1 ||
+      (artists_without_artworks && artists_without_artworks.length > 1)) {
```

Comment:
@mzikherman Following up from our offline chat, please remember to remove the redundant checks other than the use of the `group` field, seeing as Gravity already implicitly encodes those checks into `group`.